integrated_channels/integrated_channel/models.py

```diff
@@ -3,12 +3,27 @@
 """
 from __future__ import absolute_import, unicode_literals
 
+import logging
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 from model_utils.models import TimeStampedModel
 
-from enterprise.models import EnterpriseCustomer
+from enterprise.models import EnterpriseCustomer, EnterpriseCourseEnrollment
+from enterprise.utils import NotConnectedToEdX
```

Comment:
Cool, thanks for letting me know...  I'll need to rebase to get that change in, but will wait for the next round of reviews to be done first.